lib/new_relic/agent/new_relic_service.rb

```diff
@@ -489,8 +489,8 @@ def record_timing_supportability_metrics(method, start_ts, serialize_finish_ts,
       # of items as arguments.
       def record_size_supportability_metrics(method, size_bytes, item_count)
         metrics = [
-          "Supportability/invoke_remote_size",
-          "Supportability/invoke_remote_size/#{method.to_s}"
+          "Supportability/Ruby/Collector/Output/Bytes",
+          "Supportability/Ruby/Collector/#{method.to_s}/Output/Bytes"
```

Comment:
It looks like it's left over from the previous code, but there's no need for the `#to_s` here. The use of `#{}` will automatically see `#to_s` called on the object being interpolated, so our extra `#to_s` call is redundant. I'm not sure if Ruby actually ends up invoking `#to_s` twice, but it's definitely ok to remove ours and let `#{}` do its thing.

Reply:
Thanks for pointing this out, @fallwith! I took this as an opportunity to remove the other redundant #to_s calls in this file.